Source/Core/VideoBackends/D3D/D3DShader.cpp

```diff
@@ -54,10 +54,12 @@ bool CompileVertexShader(const std::string& code, D3DBlob** blob)
 		file << code;
 		file.close();
 
-		PanicAlert("Failed to compile vertex shader!\nThis usually happens when trying to use Dolphin with an outdated GPU or integrated GPU like the Intel GMA series.\n\nIf you're sure this is Dolphin's error anyway, post the contents of %s along with this error message at the forums.\n\nDebug info (%s):\n%s",
-						filename.c_str(),
-						D3D::VertexShaderVersionString(),
-						(const char*)errorBuffer->GetBufferPointer());
+		PanicAlertT("Failed to compile vertex shader!\n"
+		            "This usually happens when trying to use Dolphin with an outdated GPU or integrated GPU like the Intel GMA series.\n\n"
```

Comment:
Then I'll change it back to a PanicAlert and remove that line. It sounds like this error wouldn't be triggered for reasons that users can control.